src/client/ui/EditorContainer.js

```diff
@@ -368,9 +374,38 @@ class EditorContainer extends Component {
     if (this.props.editor.sceneInfo.uri === uri) return;
     if (!this.confirmSceneChange()) return;
 
-    this.props.editor.openRootScene(uri).catch(e => {
-      console.error(e);
-    });
+    this.props.editor
+      .openRootScene(uri)
+      .then(scene => {
+        document.title = `Hubs Editor - ${scene.name}`;
+      })
+      .catch(e => {
+        this.setState({
+          systemMessage: {
```

Comment:
This should use Robert's new `DialogContext` with the `openModal` state property. In fact, I think you can just use the `OptionDialog` and ditch `SystemMessageModalContainer`